### PR TITLE
add instructions to make submissions through cli on frontend v2

### LINKED ISSUE #3348: [Frontend V2] Add option to submit using evalai cli
### Description

In frontend v1 we show the option to make a submission using EvalAI-cli. We should add the same feature in frontend v2. Here's how the option looks in frontend v1.

Screen Shot 2021-03-24 at 12 42 02 PM

### PR DESCRIPTION
**Pull request by burnerlee**
resolves #3348

### CODE REVIEW

**Comment by burnerlee** [comment]
@Ram81 please review

**Comment by Ram81** [comment]
@burnerlee can you share a screenshot of the submit page

**Comment by burnerlee** [comment]
@Ram81 uploaded

**Comment by Ram81** [comment]
@burnerlee can you please fix the alignment of `public` and `private` tags in the instructions
Screen Shot 2021-03-30 at 12 09 13 PM

**Comment by burnerlee** [comment]
@Ram81 this alignment issue was also there in the frontend_v1.. fixing both in this PR.

**Comment by burnerlee** [comment]
Screenshot_2021-03-31_08-42-16

File: frontend/src/css/modules/submission.scss [hunk 1]
```diff
@@ -111,3 +111,9 @@ md-radio-button .md-on {
 	padding-right: 30px;
 	padding-left: 30px;
 }
+
+.center-content{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+  }
```
Comment (Ram81) [inline frontend/src/css/modules/submission.scss:119]: Please remove extra spaces before `}`

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Comment by burnerlee** [comment]
@Ram81 removed the whitespace

**Review by Ram81 — APPROVED** [review]
LGTM

**Pull request merged**